packages/office-addin-mock/src/officeMockObject.ts

```diff
@@ -82,6 +82,14 @@ export class OfficeMockObject {
     }
   }
 
+  private loadAllProperties() {
+    this.properties.forEach((_, propertyName: string) => {
+      this.properties.get(propertyName)?.loadCalled();
+      this.makePropertyCallable(propertyName);
+      this.properties.get(propertyName)?.loadAllProperties();
```

Comment:
Doesn't this create the same recursion that load scalar was doing that you removed?  We only want to go one layer down and it seems like the loop itself accomplishes that by itself doesn't it?

Reply:
For some reason I though `load("*");` would recursively load properties, but it doesn't (just tested again).
Reverted it so it only loads a single layers